src/components/cards/Card.js

```diff
@@ -116,17 +124,17 @@ export default function Card({
         <BelowContent>
           {/* <TimeStyled>{diffTime}</TimeStyled> */}
           <p onClick={toggleComments}>
-            {comments.length > 1 && comments.length + " "}
+            {commentChk.length > 1 && commentChk.length + " "}
 
-            {comments.length === 0
+            {commentChk.length === 0
```

Comment:
This stacked ternary operator looks confusing to me. Is there a way to clarify the functionality?